vlib/v/checker/tests/if_match_result.out

```diff
@@ -19,24 +19,23 @@ vlib/v/checker/tests/if_match_result.vv:7:3: error: `if` expression requires an
     8 | }
     9 |
 vlib/v/checker/tests/if_match_result.vv:11:3: error: assignment mismatch: 1 variable(s) 0 value(s)
-    9 |
+    9 | 
    10 | // void results
    11 | _ = match 4 {
       |   ^
-   12 |     1 {println('')}
-   13 |     else {exit(0)}
-vlib/v/checker/tests/if_match_result.vv:16:2: error: Unable to have an expression with void type in last statement
+   12 |     1 { println('') }
+   13 |     else { exit(0) }
+vlib/v/checker/tests/if_match_result.vv:16:2: error: unable to have an expression with void type to be used later in last statement
```

Comment:
```suggestion
vlib/v/checker/tests/if_match_result.vv:16:2: error: final expression may not have void return type
```

Reply:
It is already changed, not sure why github still shows the old version.